requirements.txt

```diff
@@ -0,0 +1,8 @@
+altair
+dash==2.1.0
+vega_datasets
+dash_bootstrap_components
```

Comment:
Can you also add os,I added it after using for reading the current directory.

Reply:
Sure